python/Produce_Config_File.py

```diff
@@ -0,0 +1,136 @@
+text_file.write("\tCanv_Margin_Lf = '%s';\n"%(args.CanvMarginLf))
+text_file.write("\tCanv_Margin_Rt = '%s';\n"%(args.CanvMarginRt))
+text_file.write("\tCanv_Name = '%s';\n"%(args.CanvName))
+text_file.write("\tCanv_Plot_Type = '%s';\n"%(args.CanvPlotType))
+text_file.write("\tCanv_Range_X = '%s';\n"%(args.CanvRangeX))
+text_file.write("\tCanv_Range_Y = '%s';\n"%(args.CanvRangeY))
+text_file.write("\tCanv_Title_Offset_X = '%s';\n"%(args.CanvTitleOffsetX))
+text_file.write("\tCanv_Title_Offset_Y = '%s';\n"%(args.CanvTitleOffsetX))
+text_file.write("\tCanv_Title_X = '%s';\n"%(args.CanvTitleX))
+text_file.write("\tCanv_Title_Y = '%s';\n"%(args.CanvTitleY))
+
+for filelist in args.file:
+	workbook = xlrd.open_workbook(filelist)
+	ws = workbook.sheet_by_index(args.SelectSheetNum)
+	GEcharacter=int(filelist.index('GE'))
+	legEntry=filelist[GEcharacter:GEcharacter+18]
+	Marker_Style = int(20+DrawArg)
+
+	text_file.write("\t\t[BEGIN_PLOT]\n" )
+	text_file.write("\t\t\tPlot_Color = '%s';\n"%(CyclicColor(DrawArg)))
+	text_file.write("\t\t\tPlot_LegEntry = '%s';\n"%(legEntry))
+	text_file.write("\t\t\tPlot_Line_Size = '%s';\n"%(args.PlotLineSize))
+	text_file.write("\t\t\tPlot_Line_Style = '%s';\n"%(args.PlotLineStyle))
+	text_file.write("\t\t\tPlot_Marker_Size = '%s';\n"%(args.PlotMarkerSize))
+	text_file.write("\t\t\tPlot_Marker_Style = '%s';\n"%(Marker_Style))
+	text_file.write("\t\t\tPlot_Name = '%s';\n"%(str(legEntry)))
+	text_file.write("\t\t\t[BEGIN_DATA]\n" )
+	text_file.write("\t\t\tVAR_INDEP,VAR_DEP,VAR_INDEP_ERR,VAR_DEP_ERR\n" )
+
+	for row in range(args.SelectRowStart,args.SelectRowEnd):
+		if bool(args.YaxisScale):
+			Y= float(ws.cell_value(row,args.SelectColumnY)/1000)
+			pass
+		else:
+			Y= ws.cell_value(row,args.SelectColumnY)
+			pass
+
+		if bool(args.SetErrX):
+			ErrX=ws.cell_value(row,args.SelectColumnErrX)
+			pass
+		else:
+			ErrX=0.
+			pass
+
+		if bool(args.SetErrY):
+			ErrY=ws.cell_value(row,args.SelectColumnErrY)
+			pass
+		else:
+			ErrY=0.
+			pass
+
+		X= ws.cell_value(row,args.SelectColumnX) 
+		text_file.write("\t\t\t"+str(X)+","+str(Y)+","+str(ErrX)+","+str(ErrY)+"\n")
+		pass
+
+	text_file.write("\t\t\t[END_DATA]\n" )
+
+	if bool(args.Fit):
+		text_file.write("\t\t\t[BEGIN_FIT]\n" )
+		text_file.write("\t\t\tFit_Color = '%s';\n"%(CyclicColor(DrawArg)))
+		text_file.write("\t\t\tFit_Formula = '%s';\n"%(args.FitFormula))
+		text_file.write("\t\t\tFit_LegEntry = 'Fit_%s';\n"%(legEntry))
+		text_file.write("\t\t\tFit_Line_Size = '%s';\n"%(args.FitLineSize))
+		text_file.write("\t\t\tFit_Line_Style = '%s';\n"%(args.FitLineStyle))
+		text_file.write("\t\t\tFit_Name = 'Fit_%s';\n"%(legEntry))
+		text_file.write("\t\t\tFit_Option = '%s';\n"%(args.FitOption))
+		text_file.write("\t\t\tFit_Param_IGuess = '%s';\n"%(args.FitParamIGuess))
+		text_file.write("\t\t\tFit_Perform = '%s';\n"%(args.FitPerform))
+		text_file.write("\t\t\tFit_Range = '%s';\n"%(args.FitRange))
```

Comment:
lines 119-128 insert an additional `\t` character so that the text appears nested w.r.t. the header for easier readability.

Reply:
done..